test/skate_web/views/layout_view_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule SkateWeb.LayoutViewTest do
+  use SkateWeb.ConnCase
+  import Test.Support.Helpers
+
+  alias SkateWeb.LayoutView
+
+  describe "record_fullstory?/0" do
+    test "returns true if the :record_fullstory env is true" do
+      reassign_env(:skate, :record_fullstory, true)
+
+      assert LayoutView.record_fullstory?()
+    end
+
+    test "returns false if the :record_fullstory env is false" do
+      reassign_env(:skate, :record_fullstory, false)
+
+      refute LayoutView.record_fullstory?()
```

Comment:
praise: I'd not seen/noticed `refute/` before this, this is neat!

Reply:
Credit to @arkadyan who wrote these tests originally, I'm just restoring them!